### DISCUSSION TITLE
VPN seulement via github/VPN only for github

### DISCUSSION THREAD

**Discussion by detobel36** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#863] [Q&A]
[FR]
Bonjour,

Je voulais savoir pourquoi le système de VPN n'était pas présent dans les release mais bien sur github ?
https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/blob/dev/.github/workflows/release.yml#L84

Merci d'avance pour les réponses

------------------------------
[EN]
Hello,

I wanted to know why the VPN system was only present on Github and not in the release ?
https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/blob/dev/.github/workflows/release.yml#L84

Thanks in advance for the replies

**Comment by darodi** [comment]
Il y a bien les  2 dans les releases


https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/blob/f391a23fc1a9aecf8705cac3efa48e78113bd9cb/.github/workflows/release.yml#L81-L82


https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/blob/f391a23fc1a9aecf8705cac3efa48e78113bd9cb/.github/workflows/release.yml#L118-L124


https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/releases/tag/v0.2.37

![image](https://user-images.githubusercontent.com/4682830/169888927-147a7b5d-8539-4cf4-a853-9d2fe5e9063d.png)


- plugin.video.catchuptvandmore-no-openvpn.zip  2.46 MB
- plugin.video.catchuptvandmore.zip  2.47 MB


![image](https://user-images.githubusercontent.com/4682830/169888653-3a90b22b-5de3-4011-af92-a1b5316d293f.png)

**Comment by detobel36** [comment]
Okay, je viens de comprendre pourquoi je ne l'avais plus après une "mise à jour"...
Il semblerait qu'une version de CatchUpTv soit sur un autre repos (un repo Kodi) et que cette version ai un "+matrix1..". Ce que kodi considère comme une "mise à jour" :confused: 

J'ai "downgrade" (simplement installé la version présente sur le repos de CatchUpTv) et j'ai bien les paramètre VPN :wink: 

Merci

**Comment by joaopa00** [comment]
Le repo officiel de Kodi n'accepte pas la présence de VPN. pour cette raison, seule la version du repo local de Catchuptvandmore accepte les VPN.
